app/routes.js

```diff
@@ -42,14 +42,11 @@ export const routes = (authProvider) => {
     res.render('pages/dashboard/template.njk', addContext({ context, user: req.user }));
   }));
 
-  router.get('/organisation/neworder', authProvider.authorise({ claim: 'ordering' }), withCatch(authProvider, async (req, res) => {
-    const context = getNewOrderPageContext();
-    res.render('pages/order-task-list/template.njk', addContext({ context, user: req.user }));
-  }));
-
   router.get('/organisation/:orderId', authProvider.authorise({ claim: 'ordering' }), withCatch(authProvider, async (req, res) => {
+    const accessToken = extractAccessToken({ req, tokenType: 'access' });
     const { orderId } = req.params;
-    res.status(200).send(`existing order ${orderId} page`);
+    const context = await getOrderTaskListPageContext({ accessToken, orderId });
```

Comment:
also don't like order in this name - it is the order form about orders and there is only 1 task list

Reply:
ok yeah fair enough. I'll update that